tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -293,8 +300,9 @@ def testTelemetryValidatorDetectsStringAsInvalidNumber(self):
     self.assertLen(errors, 1)
 
   def testTelemetryValidatorDetectsBooleanAsInvalidNumber(self):
+    is_udmi = False
     validator = telemetry_validator.TelemetryValidator(GOOD_ENTITIES_1, 1,
-                                                       _NullCallback)
+                                                       is_udmi, _NullCallback)
```

Comment:
you could do it here directly, `is_udmi=False` there is no need to declare it before.

```
    validator = telemetry_validator.TelemetryValidator(GOOD_ENTITIES_1, 1,
                                                       _NullCallback)
                                                       is_udmi=False, _NullCallback)
                                                       ```